src/components/ExperimentalFilters/Root.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactNode } from "react";
+import { Box, Text } from "..";
+
+import { FilterContext } from "./context";
+import { FilterEvent, useEventEmitter } from "./useEvents";
+import { Row } from "./Row";
+import { LeftOperatorOption } from "./EventEmitter";
+
+export type ExperimentalFiltersProps = {
+  value: Array<Row | string>;
+  leftOptions: LeftOperatorOption[];
+  children?: ReactNode;
+  onChange: (event: FilterEvent["detail"]) => void;
+};
+
+const locale: Record<string, string> = {
+  WHERE: "Where",
+  AND: "And",
+  OR: "Or",
+};
```

Comment:
**question**: will this be translated?

Reply:
those are default translations - in order for those to be translated dashboard dev will have to provide `locale` prop to `Filter`